.test-infra/pubsub/README.md

```diff
@@ -0,0 +1,36 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Overview
+
+This folder contains Python scripts to create a Pub/Sub topic under
+the GCP project `apache-beam-testing` and test the topic.
```

Comment:
Ok - I think I'm fine leaving as is, but it would be good to incorporate this more directly into tests or benchmarks _somewhere_. I can generally see the utility of this, my concern is just that nobody will be aware of it.

Reply:
This will be consumed by `dataflow-ml-starter` first and later we can use this for other tests/examples/notebooks/tutorials. The current process is also manual, which can be improved later for sure.